### ISSUE TITLE
Social media links lack discernible names

### ISSUE THREAD

**Issue by jdleonard** [Drupal-NYC/drupalcampnyc#85]
For example:
```html
<a href="https://www.twitter.com/DrupalCampNYC" aria-label="Follow us on Twitter" title="Follow us on Twitter">
        <span class="fa fa-twitter fa-2x"></span>
      </a>
```
does not have any text within the `a` tag.

Reference: https://web.dev/link-name/?utm_source=lighthouse&utm_medium=devtools
